terraform/container-registry.tf

```diff
@@ -0,0 +1,64 @@
+resource "aws_ecs_cluster" "data_refinery" {
+  name = "data-refinery"
+}
+
+resource "aws_iam_role" "ecs_service_role" {
+  name = "data-refinery-ecs-service"
+
+  assume_role_policy = <<EOF
+{
```

Comment:
It would be good to have some comments before this describing what this iam policy is doing exactly.